share/autostart

```diff
@@ -21,7 +23,7 @@ Mod=Mod1    # Use alt as the main modifier
 hc keybind $Mod-Shift-q quit
 hc keybind $Mod-Shift-r reload
 hc keybind $Mod-Shift-c close
-hc keybind $Mod-Return spawn xterm
+hc keybind $Mod-Return spawn $TERMINAL
```

Comment:
You could simply use `${TERMINAL:-xterm}` here and drop the other change - see [bash parameter expansion](http://wiki.bash-hackers.org/syntax/pe#use_a_default_value)